_episodes/02-vcs.md

```diff
@@ -144,38 +142,36 @@ control of your digital research artifacts and notes.
 {: .challenge}
 
 
-## 3rd party services
+## Third-party services
 
-As you have learned in [Remotes in GitHub](http://swcarpentry.github.io/git-novice/07-github/)
-section of the [Software Carpentry Git course](http://swcarpentry.github.io/git-novice/),
-[github] website provides you a public (or private) storage for your Git repositories on the web.
-GitHub website also allows 3rd-party websites to interact with your repositories 
+As you have learned in the [Remotes in GitHub](http://swcarpentry.github.io/git-novice/07-github/) section of the [Software Carpentry Git course](http://swcarpentry.github.io/git-novice/), the [GitHub] website provides you with public (or private) storage for your Git repositories on the web.
+The GitHub website also allows third-party websites to interact with your repositories 
 to provide additional services, typically in a response to your submission of new changes
 to your repositories. Visit [GitHub Marketplace](https://github.com/marketplace) for an
 overview of the vast collection of such additional services.  Some services are free,
-some are "pay-for-service".  Students could benefit from obtaining a 
+some are "pay-for-service". Students can benefit from obtaining a 
 [Student Developer Pack](https://education.github.com/pack) to gain free access to
 some services which otherwise would require a fee.
 
 ### Continuous integration
 
 There is a growing number of online services providing
-**continuous integration** ([CI]) services.  Although free tier unlikely to provide
-you with sufficient resources to carry out entire data analysis on your data,
-you are encouraged to use CIs to verify your code reproducible and correct execution
+**continuous integration** ([CI]) services.  Although the free tier is unlikely to provide
+you with sufficient resources to carry out entire data analyses on your data,
+you are encouraged to use CIs to verify your code’s reproducibility and correct execution
```

Comment:
well, it is more of "results reproducibility" or that code execution is reproducible and correct